### PR TITLE
[Feature][JsonSplit] Fix taskInstance

### PR DESCRIPTION
**Pull request by brave-lee**
#4417

### CODE REVIEW

File: dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/worker/sql/SqlExecutorTest.java [hunk 1]
```diff
@@ -114,16 +112,16 @@ private void sharedTestSqlTask(String nodeName, String taskAppId, String tenantC
...
 
         // custom logger
-        Logger taskLogger = LoggerFactory.getLogger(LoggerUtils.buildTaskId(LoggerUtils.TASK_LOGGER_INFO_PREFIX,
-                taskInstance.getProcessDefinitionId(),
-                taskInstance.getProcessInstanceId(),
-                taskInstance.getId()));
+//        Logger taskLogger = LoggerFactory.getLogger(LoggerUtils.buildTaskId(LoggerUtils.TASK_LOGGER_INFO_PREFIX,
+//                taskInstance.getProcessDefinitionId(),
+//                taskInstance.getProcessInstanceId(),
+//                taskInstance.getId()));
```
Comment (wen-hemin) [inline dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/worker/sql/SqlExecutorTest.java]: It can be deleted here

**Review by wen-hemin — CHANGES_REQUESTED** [review]
Please fix it

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/LoggerUtils.java [hunk 2]
```diff
@@ -67,20 +67,15 @@ private LoggerUtils() {
...
     public static String buildTaskId(String affix,
-                                     int processDefId,
                                      int processInstId,
                                      int taskId) {
         // - [taskAppId=TASK_79_4084_15210]
-        return String.format(" - %s%s-%s-%s-%s]", TASK_APPID_LOG_FORMAT, affix,
-                processDefId,
-                processInstId,
-                taskId);
+        return String.format(" - %s%s-%s-%s]", TASK_APPID_LOG_FORMAT, affix, processInstId, taskId);
```
Comment (wen-hemin) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/LoggerUtils.java:78]: It is suggested to modify it to be consistent with the log path rule.
It is convenient to debug through log.

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteProcessor.java [hunk 3]
```diff
@@ -133,7 +133,6 @@ public void process(Channel channel, Command command) {
         setTaskCache(taskExecutionContext);
         // custom logger
         Logger taskLogger = LoggerFactory.getLogger(LoggerUtils.buildTaskId(LoggerUtils.TASK_LOGGER_INFO_PREFIX,
-            taskExecutionContext.getProcessDefineId(),
```
Comment (wen-hemin) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteProcessor.java:136]: It is suggested to modify it to be consistent with the log path rule

**Review by wen-hemin — CHANGES_REQUESTED** [review]
Please fix.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/LoggerUtils.java (hunk 2 above)
Comment (brave-lee) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/LoggerUtils.java:78]: This is taskAppId, not log path

**Review by wen-hemin — APPROVED** [review]
+1

**Pull request merged**